pyvrp/cpp/search/primitives.cpp

```diff
@@ -85,3 +85,49 @@ pyvrp::Cost pyvrp::search::removeCost(Route::Node *U,
 
     return deltaCost;
 }
+
+pyvrp::Cost pyvrp::search::inplaceCost(Route::Node *U,
```

Comment:
Not the PR for this, but this operator may also be interesting for PCVRP because we only remove/insert clients now. What do you think?